datahub-web-react/src/app/entity/shared/tabs/Documentation/DocumentationTab.tsx

```diff
@@ -18,18 +18,27 @@ import { useEntityData, useRefetch, useRouteToTab } from '../../EntityContext';
 const DocumentationContainer = styled.div`
     margin: 0 auto;
     padding: 40px 0;
-    max-width: 550px;
+    max-width: calc(100% - 10px);
+    margin: 32px;
 `;
 
 export const DocumentationTab = () => {
-    const { entityData } = useEntityData();
+    const { urn, entityData } = useEntityData();
     const refetch = useRefetch();
     const description = entityData?.editableProperties?.description || entityData?.properties?.description || '';
     const links = entityData?.institutionalMemory?.elements || [];
+    const localStorageDictionary = localStorage.getItem('editedDescriptions');
 
     const routeToTab = useRouteToTab();
     const isEditing = queryString.parse(useLocation().search, { parseBooleans: true }).editing;
 
+    React.useEffect(() => {
```

Comment:
We already imported 'React' at the top of file. You can just use "useEffect"

Reply:
Removed and commited.